addons/binding/org.openhab.binding.nest/ESH-INF/thing/smoke-detector.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="nest" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="smoke_detector" listed="false">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+
+		<label>Nest Protect</label>
+		<description>Provides the nest protect layer to talk to nest</description>
+
+		<channels>
+			<channel id="ui_color_state" typeId="uicolorstate">
+				<label>UI Color</label>
+				<description>The current color of the ring on the smoke detector</description>
+			</channel>
+			<channel id="battery" typeId="battery">
```

Comment:
instead of your own "battery" channel type, you should probably better use the system-wide "system.low-battery".